app/presenters/dttp/trainee_presenter.rb

```diff
@@ -27,6 +27,7 @@ def contact_params
         "mobilephone" => trainee.phone_number,
         "dfe_EthnicityId@odata.bind" => "/dfe_ethnicities(#{dttp_ethnicity_id})",
         "dfe_DisibilityId@odata.bind" => "/dfe_disabilities(#{dttp_disability_id})",
+        "parentcustomerid_account@odata.bind" => "/accounts(#{trainee.provider.dttp_id})",
```

Comment:
We could just do `provider.dttp_id` here as we're delegating missing to `trainee`. In fact, all of the `trainee.` are redundant here.